build.sbt

```diff
@@ -15,49 +15,47 @@
  */
 
 import sbt.Keys.{baseDirectory, parallelExecution, unmanagedClasspath}
-import sbt._
+import sbt.*
 import uk.gov.hmrc.DefaultBuildSettings.{addTestReportOption, defaultSettings}
```

Comment:
The only imports needed here are:
`import sbt.*
import uk.gov.hmrc.DefaultBuildSettings`
The rest can be removed